src-docs/src/views/suggest/saved_queries.js

```diff
@@ -0,0 +1,165 @@
+
+import { GlobalFilterBar } from './global_filter_bar';
+import GlobalFilterOptions from './global_filter_options';
+import HashtagPopover from './hashtag_popover';
+
+const shortDescription = 'This is the description';
+
+const sampleItems = [
+  {
+    type: { iconType: 'kqlField', color: 'tint4' },
+    label: 'Field sample',
+    description: shortDescription,
+  },
+  {
+    type: { iconType: 'kqlValue', color: 'tint0' },
+    label: 'Value sample',
+    description: shortDescription,
+  },
+  {
+    type: { iconType: 'kqlSelector', color: 'tint2' },
+    label: 'Conjunction sample',
+    description: shortDescription,
+  },
+  {
+    type: { iconType: 'kqlOperand', color: 'tint1' },
+    label: 'Operator sample',
+    description: shortDescription,
+  },
+  {
+    type: { iconType: 'search', color: 'tint8' },
+    label: 'Recent search',
+  },
+  {
+    type: { iconType: 'save', color: 'tint3' },
+    label: 'Saved search',
+  },
+];
+
+export default class extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      status: 'unchanged',
+      value: '',
+      hideDatepicker: false,
+      filters: [
+        {
+          id: 'filter0',
+          field: '@tags.keyword',
+          operator: 'IS',
+          value: 'value',
+          isDisabled: false,
+          isPinned: true,
+          isExcluded: false,
+        },
+        {
+          id: 'filter1',
+          field: '@tags.keyword',
+          operator: 'IS',
+          value: 'value',
+          isDisabled: true,
+          isPinned: false,
+          isExcluded: false,
+        },
+        {
+          id: 'filter2',
+          field: '@tags.keyword',
+          operator: 'IS NOT',
+          value: 'value',
+          isDisabled: false,
+          isPinned: true,
+          isExcluded: true,
+        },
+        {
+          id: 'filter3',
+          field: '@tags.keyword',
+          operator: 'IS',
+          value: 'value',
+          isDisabled: false,
+          isPinned: false,
+          isExcluded: false,
+        },
+      ],
+    };
+  }
+
+  onFieldFocus() {
+    this.setState({
+      hideDatepicker: true,
+    });
+  }
+
+  onFieldBlur() {
+    this.setState({
+      hideDatepicker: false,
+    });
+  }
+
+  getInputValue(val) {
+    this.setState({
+      value: val,
+    });
+  }
+
+  onItemClick(item) {
+    alert(`Item [${item.label}] was clicked`);
+  }
+
+  onTimeChange() {
+    alert('Time changed');
+  }
+
+  render() {
+    const append = <EuiButtonEmpty>KQL</EuiButtonEmpty>;
+
+    return (
+      <div className="savedQueriesInput">
+        <EuiFlexGroup
+          gutterSize="s"
+          className={this.state.hideDatepicker ? 'hideDatepicker' : ''}>
+          <EuiFlexItem>
+            <EuiSuggest
+              status={this.state.status}
+              onFocus={this.onFieldFocus.bind(this)}
+              onBlur={this.onFieldBlur.bind(this)}
+              prefix={<HashtagPopover value={this.state.value} />}
```

Comment:
And then this prop just gets renamed to `prepend`
```suggestion
              prepend={<HashtagPopover value={this.state.value} />}
```